test/reporters.js

```diff
@@ -1427,7 +1614,7 @@ describe('Reporter', () => {
                 '<td class="source"><div>    while ( </div><div class="miss false" data-tooltip>value ) </div><div>{</div></td>']);
             expect(output, 'missed original line not included').to.contains([
                 '<tr id="while.js__14" class="miss">',
-                '<td class="source" data-tooltip>        value &#x3D; false;</td>']);
+                '<td class="source" data-tooltip>        value = false;</td>']);
```

Comment:
This fails with feature flag off but passes with feature flag on.

It is possible to wrap this in the feature flag as well, but not sure it is worth the effort.